integration-tests/src/test/java/com/ozonehis/eip/odoo/openmrs/ServiceRequestToSaleOrderIntegrationTest.java

```diff
@@ -82,8 +92,15 @@ public void shouldVerifySaleOrderRoutes() {
     }
 
     @Test
-    @DisplayName("Should create sale order in Odoo given medication request bundle.")
+    @DisplayName("Should create sale order with Patient Weight in Odoo given service request bundle.")
     public void shouldCreateSaleOrderInOdooGivenServiceRequestBundle() {
+        // Setup
+        mockOpenmrsFhirServer();
```

Comment:
Move mock initialization to the `@BeforeEach` method and remove it from individual test methods